math/matrix/inc/TVectorT.h

```diff
@@ -27,15 +27,15 @@
 template<class Element> class TVectorT : public TObject {
 
 protected:
-   Int_t    fNrows;                // number of rows
-   Int_t    fRowLwb;               // lower bound of the row index
-   Element *fElements;             //[fNrows] elements themselves
+   Int_t    fNrows{0};                // number of rows
```

Comment:
IIUC, the link is focused towards objects, and indeed when initializing object it will be better. My comment is about initializing scalar types.